src/Illuminate/Database/Eloquent/Concerns/HasAttributes.php

```diff
@@ -1612,9 +1612,16 @@ protected function isEnumCastable($key)
      */
     protected function isClassDeviable($key)
     {
-        return $this->isClassCastable($key) &&
-            method_exists($castType = $this->parseCasterClass($this->getCasts()[$key]), 'increment') &&
-            method_exists($castType, 'decrement');
+        if (! $this->isClassCastable($key)) {
+            return false;
+        }
+
+        $castType = $this->parseCasterClass($this->getCasts()[$key]);
+        if (method_exists($castType, 'castUsing')) {
+            $castType = $castType::castUsing([]);
```

Comment:
Wouldn't calling the "castUsing" method with an empty array disregards any given cast parameters?